situp-core/src/integrationTest/java/com/amazon/situp/integration/EndToEndIntegrationTest.java

```diff
@@ -0,0 +1,194 @@
+package com.amazon.situp.integration;
+
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+import com.amazon.situp.plugins.sink.elasticsearch.ConnectionConfiguration;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.UUID;
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.HttpData;
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.RequestHeaders;
+import com.linecorp.armeria.common.SessionProtocol;
+import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest;
+import io.opentelemetry.proto.common.v1.AnyValue;
+import io.opentelemetry.proto.common.v1.KeyValue;
+import io.opentelemetry.proto.resource.v1.Resource;
+import io.opentelemetry.proto.trace.v1.InstrumentationLibrarySpans;
+import io.opentelemetry.proto.trace.v1.ResourceSpans;
+import io.opentelemetry.proto.trace.v1.Span;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.search.SearchHits;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EndToEndIntegrationTest {
+
+    private static final Random RANDOM = new Random();
+    private static final List<Span.SpanKind> SPAN_KINDS =
+            Arrays.asList(Span.SpanKind.CLIENT, Span.SpanKind.CONSUMER, Span.SpanKind.INTERNAL, Span.SpanKind.PRODUCER, Span.SpanKind.SERVER);
+    private static final String INDEX_NAME = "otel-v1-apm-span-000001";
+
+    @Test
+    public void testPipelineEndToEnd() throws IOException, InterruptedException {
+        //Send data to otel trace source
+        final ExportTraceServiceRequest exportTraceServiceRequest1 = getExportTraceServiceRequest(
+                getRandomResourceSpans(10)
+        );
+
+        final ExportTraceServiceRequest exportTraceServiceRequest2 = getExportTraceServiceRequest(
+                getRandomResourceSpans(10)
+        );
+
+        sendExportTraceServiceRequestToSource(exportTraceServiceRequest1);
+        sendExportTraceServiceRequestToSource(exportTraceServiceRequest2);
+
+        //Wait for data to flow through pipeline and be indexed by ES
+        Thread.sleep(2000);
```

Comment:
I think now we have better strategy for this:

https://github.com/opendistro-for-elasticsearch/simple-ingest-transformation-utility-pipeline/blob/8c592850813901b0abb5666f7249d8b1720a14c6/situp-plugins/elasticsearch/src/test/java/com/amazon/situp/plugins/sink/elasticsearch/ElasticsearchSinkIT.java#L242

checking if it works

Reply:
Made a patching PR with the relevant change to merge into this one.